pkg/wallet/token.go

```diff
@@ -38,7 +38,7 @@ func NewToken(tokenHash util.Uint160, name, symbol string, decimals int64) *Toke
 func (t *Token) MarshalJSON() ([]byte, error) {
 	m := &tokenAux{
```

Comment:
Maybe we don't need this struct at all now?

Reply:
We use this struct in UnmarshalJSON too because it does not contains `Address` and we have to fill this field. Unmarshalling into &tokenAux lets us avoid recursion, so perhaps we still need it?

Reply:
`json:"-"`? And maybe this structure shouldn't have an `Address` field, but rather an `Address()` method (and maybe `address` field as a cache, though I doubt it's worth it, this structure is never in the hot path).

Reply:
Done (added Address() method)